### ISSUE TITLE
Ability to have the process keep working while you walk away, with an auto prompt like "Keep integrating and improving the app"

### ISSUE THREAD

**Issue by mediafill** [stackblitz-labs/bolt.diy#826]
Ability to have the process keep working while you walk away, with an auto prompt like "Keep integrating and improving the app"

Would love to be able to walk away or work on something else while bolt keeps plugging away. Maybe a button that says "Automate" and when you click it, it would ask for the number of turns (EG:10). It would then auto prompt "Keep integrating and improving the app" or whatever prompt the user sets.

**Comment by mediafill** [comment]
Then how about a separate LLM (like 4o checking Sonnet, or vice versa) checking the progress and directing the agent what to do by sending it as a prompt?

**Comment by chrismahoney** [comment]
Seems like it could be more effective to allow opt-in of “I’m feeling lucky” behavior, with some ability to specify how far it should go/whether it should hit the brakes on the first error. It’s an interesting solution to a coffee break where you’re just trying something out.

**Comment by Ravenmoray** [comment]
if you use o1 you can tell it to loop but this is useful to have

**Comment by aliasfoxkde** [comment]
Yeah, in theory this is what we all want. But right now, you have to babysit the AI too much.
